lib/pollTravis.js

```diff
@@ -138,11 +140,14 @@ function pollTravisBuildBySha (options, checkNumber) {
       const lastState = lastBuildForCommit.state
 
       if (lastState === 'passed') {
-        return createGhComment(`[Travis build passed](https://travis-ci.org/${options.owner}/${options.repoName}/builds/${lastBuildForCommit.id}) :+1:`)
+        return createGhStatus('success', lastBuildForCommit.id, 'all tests passed')
       } else if (lastState === 'failed') {
-        return createGhComment(`[Travis build failed](https://travis-ci.org/${options.owner}/${options.repoName}/builds/${lastBuildForCommit.id}) :-1:`)
+        return createGhStatus('failure', lastBuildForCommit.id, 'build failure')
       } else if (~['created', 'started'].indexOf(lastState)) {
         console.log(`* ${prInfo} "${lastState}" build found, will do check #${checkNumber + 1} in 30 seconds`)
+        if (checkNumber === 1) {
```

Comment:
Hmm, sure. I didn't because it was a bit spammy for the logs, but should be fine.